### DISCUSSION TITLE
Fault Queue Shows "?" in Messages Column

### DISCUSSION THREAD

**Discussion by njsharpe** [rabbitmq/rabbitmq-server#9795] [Questions]
Hello everyone!

I am new to RabbitMQ and trying to get some queues set up here. I have an exchange (we can call it Ruby), which is attached to two queues; Ruby.Staging and Ruby.Staging.Fault. For whatever reason, Ruby.Staging runs as expected: it's a classic type with durable enabled and the high availability mirroring policy set. However, the Ruby.Staging.Fault is broken in some manner. It shows that durable is enabled, but is does not show any policy, is neither idle or running, and under the messages column it only shows "?" for each sub-column.

I have tried my best to understand what is going on but the short of it is I don't have enough knowledge in this system to figure it out myself. I am all ears, thanks in advance!

**System Information**
- Windows Server 2019
- RabbitMQ v3.12.4
- Erlang v26.0.2

**Comment by michaelklishin** [comment]
@njsharpe it means that `Ruby.Staging.Fault` does not have an elected leader for some reason, and it does not emit metrics that the UI could display. Server logs may contain clues.

FYI, classic mirrored queues have been officially [deprecated for a couple of years now](https://blog.rabbitmq.com/posts/2023/02/quorum-queues-migration/). Their doc guide warns you that this is a feature to be avoided [in the very first paragraph](https://rabbitmq.com/ha.html).

Classic mirrored queues have known and unfixable [failure scenarios](https://rabbitmq.com/ha.html#unsynchronised-mirrors) that would manifest themselves exactly as you are describing.

Please delete and re-declare both as quorum queues.
